### ISSUE TITLE
Waterfall order (column and row)

### ISSUE THREAD

**Issue by apastore** [griffithlab/GenVisR#317]
Thanks for this package! Would be possible to let the waterfall gene and samples be sorted by two different tested criteria, let's say clinical annotation first and "hierarchical sort of gene(default in waterfall)"  second ?

The same for the genes first let's say gene function annotations (go/kegg/Munual groups) and second by the curent defaul?

Thank a lot!

**Comment by zlskidmore** [comment]
Hi @apastore 

thanks for the suggestion! you can indirectly do this by supplying a custom order for genes and samples but i agree that this would be a nice feature. leaving this open as a possible enhancement while i think about the best way to implement this

**Comment by apastore** [comment]
Yes exactly! I am manually providing the gene and sample order. 
Thanks a lot for this package!
